examples/cryptocurrency-advanced/backend/src/transactions.rs

```diff
@@ -65,44 +66,62 @@ impl From<Error> for ExecutionError {
     }
 }
 
-transactions! {
-    /// Transaction group.
-    pub WalletTransactions {
-
-        /// Transfer `amount` of the currency from one wallet to another.
-        struct Transfer {
-            /// `PublicKey` of receiver's wallet.
-            to:      &PublicKey,
-            /// Amount of currency to transfer.
-            amount:  u64,
-            /// Auxiliary number to guarantee [non-idempotence][idempotence] of transactions.
-            ///
-            /// [idempotence]: https://en.wikipedia.org/wiki/Idempotence
-            seed:    u64,
-        }
+/// Transfer `amount` of the currency from one wallet to another.
+#[derive(Serialize, Deserialize, Clone, Debug, ProtobufConvert)]
+#[protobuf_convert("proto::Transfer")]
```

Comment:
I vaguely remember that attributes support namespaces now (like `exonum::protobuf_convert`), judging by namespaced `clippy` attributes in the code of the main crate. Please check and use the namespaced version if it's possible (you can shorten the attribute name to something like `exonum::pb()` as well).